### ISSUE TITLE
Change from Traefik to Nginx

### ISSUE THREAD

**Issue by cur1ou5soul** [kube-hetzner/terraform-hcloud-kube-hetzner#473]
Is it possible to switch from traefik to nginx?
 
Cause just setting enable_nginx=true enable_traefik=false nothing will change with terraform plan.

**Comment by captnCC** [comment]
I don't think it is possible after the cluster is created (at least via terraform). The problem is, that the ingress classes are only created while provisioning the first control plane. 
The switch from traefik to nginx would need to be done manually. This would include deleting the old traefik manifests and applying the manifests for the nginx
